src/devtools/client/inspector/changes/reducers/changes.ts

```diff
@@ -59,22 +85,23 @@ function cloneState(state = {}) {
  * @return {Object}
  *         Entry for the CSS rule created the given collection of rules.
  */
-function createRule(ruleData, rules) {
+function createRule(ruleData: Partial<Rule>, rules: RuleRecord) {
   // Append the rule data to the flattened CSS rule tree with its ancestors.
-  const ruleAncestry = [...ruleData.ancestors, { ...ruleData }];
+  const ruleAncestry = [...ruleData.ancestors!, { ...ruleData }];
```

Comment:
No idea, because I don't know what the rest of the type structure looks like here :)

I'm not trying to get _perfect_ types here.  Just "good enough" for now.